### ISSUE TITLE
Rendor never adding ( and { to tokens

### ISSUE THREAD

**Issue by StandingPadAnimations** [StandingPadAnimations/Rendor#2]
Seems like a bug involving the TempID variable when it's being checked here(when running the if.ren file):
https://github.com/StandingPadAnimations/Rendor/blob/15e528c54936720caab3cb6fcca58f4a43131466/src/Lexer.cpp#L60-L78

The output I intended would be:
```
Reached Operator 
Reached Operator 
5 b
10 if
7 PAREN
5 b
9 OR
9 OR
5 c
7 PAREN
8 BRACK // I know it won't do this in the current code because it uses the TempString variable but that's an easy change
5 hi
```
Whereas the actual output is:
```
Reached Operator 
Reached Operator 
5 b
10 if
5 b
9 OR
9 OR
5 c
7 PAREN
5 hi
```

**Comment by StandingPadAnimations** [comment]
The bracket portion of the issue has been fixed, but not the issue involving parentheses. I'll have to look in it further

**Comment by StandingPadAnimations** [comment]
This has been fixed in commit 1650e46790ac0cc55be888aa3ecaf8ccc3a74f4f
